src/posts/posts.service.ts

```diff
@@ -77,7 +83,8 @@ export class PostsService {
                authorId: userId
             }
         })
-        return post
+        const action = await this.actionsService.addAction('Create',userId,'Post',post.id,{title,description,image})
```

Comment:
here you have hadcoded strings. Store them in variables and reuse? insted of hardcode them